region/norfair/west.json

```diff
@@ -2901,6 +2901,31 @@
               ],
               "openEnd": 1
             }
+          ],
+          "canLeaveCharged": [
+            {
+              "usedTiles": 33,
+              "framesRemaining": 0,
+              "shinesparkFrames": 12,
+              "openEnd": 2,
+              "strats": [
+                {
+                  "name": "Quick Platforming",
+                  "notable": false,
+                  "requires": [
+                    "h_heatProof",
+                    {"obstaclesCleared": ["A"]},
+                    "canShinechargeMovementComplex",
+                    "canFastWalljumpClimb"
+                  ],
+                  "note": [
+                    "Move very quickly to bring a shinespark from the speedway up to the Save Room door.",
+                    "Preclear the Cacatacs, and it may help to not run while platforming if HiJump is not available."
+                  ],
+                  "devNote": "FastWalljumpClimb mainly included for difficulty placement.  Fast movement is needed for this strat."
```

Comment:
This wont be usable in the current schema, as youll need to start at door 5, so there's no way to clear A. I think its still worth including, as it should be something we address soon.